packages/frontend/editor-ui/src/components/CanvasChat/future/components/LogsOverviewPanel.vue

```diff
@@ -141,6 +141,7 @@ watch(
 						type="secondary"
 						icon="trash"
 						icon-size="medium"
+						data-test-id="clear-execution-data-button"
```

Comment:
Does this intentionally have the same data-test-id as `CanvasClearExecutionDataButton` ?

Reply:
Yes, we only show either of the two (and we'll remove the other one eventually when we'll clean up the feature flag)